ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Allocation.scala

```diff
@@ -8,9 +8,17 @@ import com.daml.ledger.client.binding.Primitive.Party
 
 private[testtool] object Allocation {
   def allocate(firstPartyCount: PartyCount, partyCounts: PartyCount*): ParticipantAllocation =
-    ParticipantAllocation(firstPartyCount +: partyCounts)
-
-  final case class ParticipantAllocation private (partyCounts: Seq[PartyCount])
+    ParticipantAllocation(firstPartyCount +: partyCounts, minimumParticipantCount = 1)
+
+  final case class ParticipantAllocation private (
+      partyCounts: Seq[PartyCount],
+      minimumParticipantCount: Int,
+  ) {
+    def expectingActualParticipantCountOfAtLeast(
```

Comment:
```suggestion
    def expectingMinimumParticipantCount(
```

Reply:
I would like to keep the word "actual" in there because we have this concept of participants where two labels might actually be the same participant. Otherwise, I like your suggestion.